### DISCUSSION TITLE
HASHIFY API down

### DISCUSSION THREAD

**Discussion by cyryl2004** [dwin/hashify#11] [General]
https://stats.uptimerobot.com/QnZl8svqB

HASHIFY API is down

![image](https://github.com/dwin/hashify/assets/6866903/bf826d10-e4cf-4d1c-9e40-e649d5ad4b99)

**Comment by Jugo01** [comment]
Did we have some updates about this issue ?

**Comment by cyryl2004** [comment]
Still down https://stats.uptimerobot.com/QnZl8svqB

śr., 2 sie 2023, 09:29 użytkownik Jugo01 ***@***.***> napisał:

**Comment by dwin** [comment]
Sorry for the inconvenience. This is a free API and a hobby project. There is no uptime guarantee and availability is based on best effort.

**Comment by cyryl2004** [comment]
Ok, thanks for the reply.
Z wyrazami szacunku
Karol Bocian


czw., 17 sie 2023 o 05:33 Darwin Smith ***@***.***>
napisał(a):
